docs/playlist/kpop.md

```diff
@@ -9,3 +9,4 @@ title: K-Pop
 | -------------- |-------------- | -------- | ------- | ------- | -------- |
 | 한국 노래 | [Spotify](https://open.spotify.com/playlist/3z5iDf37a7qsOC0LfdwuwF) | Spotify | [Anthony Lee (Mbahmo)](https://github.com/mbahmo) | List of K-pop Songs that was my favorite (Bigbang, Red Velvet, Etc) |
 | This Is BLACKPINK | [Spotify](https://open.spotify.com/playlist/37i9dQZF1DX8kP0ioXjxIA) | Spotify | [Wahyu Setyawan (wahyustwn)](https://github.com/wahyustwn) | List of BLACKPINK Songs |
+| Some | [Spotify](https://open.spotify.com/track/3jsYQw78lrxJA2ysnmOIf9?si=6700c6c4dc214ae7) | Spotify | [Rino (riyaraa)](https://github.com/riyaraa) | List of Tiktok songs Kpop |
```

Comment:
Please sort alphabetically, refer to this [https://awesome-music-playlist.vercel.app/contribution.html#add-new-playlist](https://awesome-music-playlist.vercel.app/contribution.html#add-new-playlist)